src/ssl_certman.c

```diff
@@ -89,11 +89,20 @@ WOLFSSL_CERT_MANAGER* wolfSSL_CertManagerNew_ex(void* heap)
     WOLFSSL_CERT_MANAGER* cm;
 
     WOLFSSL_ENTER("wolfSSL_CertManagerNew");
+    if (heap == NULL) {
```

Comment:
Why do we care?
%p will convert NULL to something different on at least some platforms.

Reply:
> Why do we care?

I left a comment there for this. In my case _where_ the heap was being allocated was critical to the failure I was seeing. I had two segments of memory, and the heap was unexpectedly in the smaller one.